source/extensions/filters/http/aws_request_signing/aws_request_signing_filter.cc

```diff
@@ -0,0 +1,53 @@
+#include "extensions/filters/http/aws_request_signing/aws_request_signing_filter.h"
+
+#include "envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.pb.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AWSRequestSigningFilter {
+
+FilterConfigImpl::FilterConfigImpl(HttpFilters::Common::Aws::SignerPtr&& signer,
+                                   const std::string& stats_prefix, Stats::Scope& scope)
+    : signer_(std::move(signer)), stats_(Filter::generateStats(stats_prefix, scope)) {}
+
+FilterStats Filter::generateStats(const std::string& prefix, Stats::Scope& scope) {
```

Comment:
nit: This doesn't used any private members of the class `Filter`.  I recommend making it a free function within the `AWSRequestSigningFilter`

Reply:
the pattern in other filters is to make it static, so lets keep it like that for consistency?